graphlearn/examples/tf/trainer.py

```diff
@@ -209,6 +276,48 @@ def train_and_evaluate(self, train_iterator, test_iterator, loss, test_acc, opti
     self.train(train_iterator, loss, optimizer, learning_rate, epochs, hooks, **kwargs)
     self.test(test_iterator, test_acc, hooks, **kwargs)
 
+  def save_node_embedding_bigdata(self, save_iter, save_ids, save_emb, save_file, block_max_lines, batch_size):
+      if batch_size >= block_max_lines:
```

Comment:
> This judgment is a little confusing. Do you mean total_lines <= block_max_lines?

The code compares the batch_size and block_max_lines variables. If batch_size is greater than block_max_lines, the embeddings of a batch will be saved in multiple blocks (text files). This can make the process more complex. The total_lines variable is not used except for printing information at the end.